### PR TITLE
Apply SwiftLint

### PR DESCRIPTION
**Pull request by alex-ioja-yang-cko**
Updated SwiftLint file to cover Example Frame folder.

Run `swiftlint autocorrect` at project level and commit all changes.

### CODE REVIEW

File: .swiftlint.yml [hunk 1]
```diff
@@ -1,7 +1,9 @@
 included: # paths to include during linting. `--path` is ignored if present.
```
Comment (ehab-al-cko) [inline .swiftlint.yml:1]: What are swiftlint rules?

Comment (alex-ioja-yang-cko) [reply .swiftlint.yml:1]: Good question 😂

Running it hasn't changed anything other than spacings, so for now I am tempted to class it as `working`, but you can start a separate discussion for changing the rules if you want 👍.

From [SwiftLint rules list](https://realm.github.io/SwiftLint/rule-directory.html) this configuration uses all the **Default Rules** and none of the **Opt-In Rules**. It then customises the default `identifier_name` rule to allow `id` as a valid property name (would normally be excluded on the grounds of being less than 3 characters long - default behaviour of that rule).

**Review by deepesh-vasthimal-cko — APPROVED** [review]
LGTM

**Comment by deepesh-vasthimal-cko** [comment]
@ehab-al-cko @alex-ioja-yang-cko We can also now enable / update CI / CD to check for linting.

File: .swiftlint.yml [hunk 2]
```diff
@@ -1,7 +1,9 @@
...
   - Source
```
Comment (alex-ioja-yang-cko) [inline .swiftlint.yml:2]: @deepesh-vasthimal-cko 

> @ehab-al-cko @alex-ioja-yang-cko We can also now enable / update CI / CD to check for linting.

I wouldn't rush to enforce linting on pipelines yet, its a bit of a waste of resources imo. We could instead review how we develop and see if its appropriate to get SwiftLint autocorrect inside the build scripts?

@harry-brown-cko have you done any discovery on doing the linting locally?

**Review by ehab-al-cko — APPROVED** [review]

**Pull request merged**